integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -633,9 +661,24 @@ public int release(String path, FuseFileInfo fi) {
   private int releaseInternal(String path, FuseFileInfo fi) {
     long fd = fi.fh.get();
     try {
+      CreateFileEntry<FileOutStream> ce = mCreateFileEntries.getFirstByField(ID_INDEX, fd);
+      if (!mTmpFolder.isEmpty() && ce != null) {
+        FileOutStream os = ce.getOut();
```

Comment:
No need to create FileOutStream until release
we only need Alluxio FileOutStream in release, separating them is more error-prone

Reply:
agree